### PR TITLE
#11 - token validation: compute largest column for a better token column validation

### LINKED ISSUE #11: avoid improper alignment posisitions
### Description of the issue

the current implementation just looks up certain assignment punctuators and compares their columns, but it doesn't care more about the actual position

### Steps to reproduce

Check out the following test case:

``` javascript
{
    code: [
      'var foo         = [];',
      'var blah        = [];'
    ].join('\n'),
    errors: [{
      message: 'Invalid indent!'
    }]
  }
```
### Expected behavior

The example code in the test case should raise an error as the punctuators should be placed direct after the variable declarations which requires the most columns.

### PR DESCRIPTION
**Pull request by Ma27**
### Overview

The following code style violation will be detected properly thanks to this PR:

``` javascript
var foo             = [];
var bar             = [];
```

### Related/Resolved tickets

resolves #11

### CODE REVIEW

File: test/lib/rules/var-spacing.rule.js [hunk 1]
```diff
@@ -581,5 +587,13 @@ ruleTester.run('var-spacing', rule, {
...
+  }, {
+    code: [
+      'var foo           = [];',
+      'var blah          = {};'
+    ].join('\n'),
+    errors: [{
+      message: 'The punctuator column must be directly after the longest variable!'
+    }]
   }]
 });
```
Comment (Ma27) [inline test/lib/rules/var-spacing.rule.js]: some more test cases should be added in order to check out certain edge cases
